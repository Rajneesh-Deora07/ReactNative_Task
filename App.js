/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow strict-local
 */

import React from 'react';
import ForecastWeather from './lib/Index';
import {Provider} from 'react-redux';
import {configureStore} from './store/configureStore';

const store = configureStore();

export default function App() {
  return (
    // Redux: Global Store
    <Provider store={store}>
      <ForecastWeather />
    </Provider>
  );
}
