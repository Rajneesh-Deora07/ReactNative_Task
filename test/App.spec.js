import React from 'react';
import {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import App from '../App';

describe('Store provider ------', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it('App Snapshot match', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
