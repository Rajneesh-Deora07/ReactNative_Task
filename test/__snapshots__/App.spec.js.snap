// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Store provider ------ App Snapshot match 1`] = `ShallowWrapper {}`;
